corehq/apps/export/custom_export_helpers.py

```diff
@@ -519,6 +519,7 @@ def get_context(self):
 
 def make_custom_export_helper(request, export_type, domain=None, export_id=None):
     export_type = export_type or request.GET.get('request_type', 'form')
+    # NOTE: minimal is a hack so that some domains can load the exports page properly :-(
```

Comment:
yea so this was for succeed domain. their exports were literally so big Chrome would crash when it would try to load. so to fix it i made a `minimal` version that wouldn't do some random things to make the page actual loading. mainly not sorting and not loading anything but the main custom export